.github/workflows/ci.yml

```diff
@@ -40,10 +40,13 @@ jobs:
         max-size: 2G
     - name: Build
       run: |
+        export GITHUB_ACTIONS=ON
```

Comment:
1. You should not do this. The point of CI is to test configuration that user have. You should make as few differences as possible.
2. Github Actions already define some env vars that are unique to this environment